lib/rules/react-no-render-methods.js

```diff
@@ -0,0 +1,46 @@
+const Components = require('eslint-plugin-react/lib/util/Components');
```

Comment:
Name is a little misleading since there is still a render method 🙃

Reply:
what about `react-no-multiple-render-methods`?